devdoc/umock_c_lib_requirements_with_ids.md

```diff
@@ -444,6 +444,15 @@ XX**SRS_UMOCK_C_LIB_01_148: [** If call comparison fails an error shall be indic
 
 **SRS_UMOCK_C_LIB_01_153: [** If no custom handler has beed registered for a pointer type, it shall be trated as void*. **]**
 
+### Windows types
+
+**SRS_UMOCKTYPES_WINDOWS_01_001: [** The following Windows types shall be supported out of the box, aliased to their underlying types: **]**
+
+- LONG (long)
+- LONGLONG (long long)
+- DWORD (unsigned long)
+- HRESULT (LONG)
```

Comment:
>HRESULT [](start = 2, length = 7)

Doesn't seem to cover all of the types that are actually supported

Reply:
Added all of them.